pir/cpp/client_test.cpp

```diff
@@ -72,12 +77,50 @@ TEST_F(PIRClientTest, TestProcessResponse) {
   Context()->Encoder()->encode(value, pt);
   vector<Ciphertext> ct(1);
   Encryptor()->encrypt(pt, ct[0]);
-  PIRPayload payload = PIRPayload::Load(ct);
 
-  auto result = client_->ProcessResponse(payload).ValueOrDie();
+  Response reply;
+  SaveCiphertexts(ct, reply.mutable_reply());
+
+  auto result = client_->ProcessResponse(reply).ValueOrDie();
   ASSERT_EQ(result, value);
 }
 
+TEST_F(PIRClientTest, TestRequestSerialization) {
+  int64_t value = 987654321;
+  Plaintext pt;
+  Context()->Encoder()->encode(value, pt);
+  vector<Ciphertext> ct(1);
+  Encryptor()->encrypt(pt, ct[0]);
+
+  Response reply_proto;
```

Comment:
If this is testing request serialization, why is this a response proto? Shouldn't this be a separate test?